packages/strapi-plugin-upload/admin/src/components/SortPicker/SortButton.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { themePropTypes } from 'strapi-helper-plugin';
+
+import Text from '../Text';
+
+const SortButton = styled(props => (
+  <Text as="button" fontWeight="semiBold" {...props} />
+))`
+  height: 32px;
+  padding: 0 10px;
+  line-height: 30px;
+  background-color: ${({ theme }) => theme.main.colors.white};
+  border: 1px solid ${({ theme }) => theme.main.colors.darkGrey};
+  color: ${({ theme }) => theme.main.colors.greyDark};
```

Comment:
You can set it directly in the `Text` component:

`<Text as="button" fontWeight="semiBold" color={props.isActive ? 'lightBlue':'greyDark} {...props} />`